yarn.lock

```diff
@@ -998,7 +998,7 @@ __metadata:
     "@types/source-map-support": ^0
     "@typescript-eslint/eslint-plugin": ^4.18.0
     "@typescript-eslint/parser": ^4.18.0
-    ardrive-core-js: 1.0.5
+    ardrive-core-js: ../ardrive-core-js
```

Comment:
As we don't commit the yarn cache of the local version of core, I usually avoid pushing also the `yarn.lock` as it will be overwritten anyways after we do `yarn install`. I'd say we don't need to push this file for when we are using a local version of core -unless there's another new dependency added-

Reply:
Yeah I think that is a good idea. It will keep the diff cleaner during the code review stage

Reply:
Updated the in progress Core -> CLI workflow doc to reflect your suggestion: https://ardrive.atlassian.net/wiki/spaces/ENGINEERIN/pages/298942467/Standard+ArDrive+Core+to+ArDrive+CLI+Workflow